src/app/containers/ATIAnalytics/canonical/index.test.jsx

```diff
@@ -19,6 +19,8 @@ describe('Canonical ATI Analytics', () => {
     process.env.SIMORGH_ATI_BASE_URL = atiBaseUrl;
     beacon.default = mockSendBeacon;
 
+    const { render: create, act } = testingLibrary;
```

Comment:
Not sure why we need to rename `render` to `create`, can we retain the name `render` for clarity?

Reply:
There's already a render imported at the top of the file, from `enzyme`.

Reply:
Makes sense